internal/cli/tables-to-go-cli.go

```diff
@@ -76,23 +91,48 @@ func (c columnInfo) hasTrue() bool {
 	return c.isNullable || c.isTemporal || c.isNullableTemporal || c.isNullablePrimitive
 }
 
-func createTableStructString(settings *settings.Settings, db database.Database, table *database.Table) (string, string) {
+func createTableStructString(settings *settings.Settings, db database.Database, table *database.Table) (string, string, error) {
 
 	var structFields strings.Builder
+	tableName := strings.Title(settings.Prefix + table.Name + settings.Suffix)
+	// Replace any whitespace with underscores
+	tableName = strings.Map(replaceSpace, tableName)
+	if settings.IsOutputFormatCamelCase() {
+		tableName = camelCaseString(tableName)
+	}
+
+	// Check that the table name doesn't contain any invalid characters for Go variables
+	if !validVariableName(tableName) {
+		return "", "", fmt.Errorf("Table name %q contains invalid characters", table.Name)
```

Comment:
Error strings should not be capitalized.

Reply:
resolved in 396b9e8